src/OpenLoco/Localisation/StringManager.h

```diff
@@ -147,4 +147,6 @@ namespace OpenLoco::StringManager
     string_id fromTownName(string_id stringId);
     std::pair<string_id, string_id> monthToString(MonthId month);
     int32_t internalLengthToComma1DP(const int32_t length);
+    size_t locoStrlen(const char* buffer);
+    char* locoStrcpy(char* dest, const char* src);
```

Comment:
I'm surprised this doesn't need a length. Reading the method, if there isn't a `\0` char in `src`, this'll go out of bounds which seems like a dangerous assumption to make

Reply:
It's a dangerous assumption but sadly one we need to make for now. Most of the caller's only have a pointer to memory rather than a known size.

Reply:
Oh ok, should we make an overload for one with a size? Just as an indication we should do that in future?

Reply:
Yes that would be a good idea